spec/support/utils/exercise-testing-util.js

```diff
@@ -7,13 +7,8 @@ function generateRandomRestInterval() {
 }
 
 
-const generateRandomExercise = (userId, exerciseId) => {
-    let name = generateRandomString(6);
-    let force = generateRandomEnumValue(enums.enumList.forceEnum);
```

Comment:
You should leave the random generation and maybe make another method that overloads this with the same name. I understand the need to have null fields, but making an overloaded method is the best (a separate method)

Reply:
I'm not sure the best way to implement this since Javascript doesn't support overloading

Reply:
no but it supports default parameters if a parameter isnt preset make it auto generated